hack/netpol/README.md

```diff
@@ -100,9 +98,9 @@ We are working to merge this into upstream Kubernetes, in the meanwhile, here's
 Create the policy probe tests:
 
 ```
-kubectl create clusterrolebinding netpol --clusterrole=admin --serviceaccount=kube-system:netpol
+kubectl create clusterrolebinding netpol --clusterrole=cluster-admin --serviceaccount=kube-system:netpol
 kubectl create sa netpol -n kube-system
-kubectl create -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/netpol/install.yml
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/netpol/install.yml -n kube-system
```

Comment:
I don't believe the `-n kube-system` does anything here?

Reply:
let me update this.. i forgot to remove it from here when i added the namespace in install.yaml